R/to_rmd.R

```diff
@@ -237,6 +234,12 @@ to_rmd.default <- function(block, output_dir, ...) {
   }
 }
 
+#' @method .to_rmd condition
+#' @keywords internal
+.to_rmd.condition <- function(block, output_dir, ...) {
+  conditionMessage(block)
```

Comment:
Should we wrap this around a chunk? it might look weird when used

```suggestion
  sprintf("```\n%s\n```, conditionMessage(block))
```